ui/mirage/handlers/sync.js

```diff
@@ -95,6 +96,33 @@ export default function (server) {
     };
   };
 
+  // client counts with upgrade history
+  server.get('sys/version-history', function () {
+    return {
+      data: {
+        keys: ['1.9.0', '1.9.1', '1.9.2', '1.10.1'],
+        key_info: {
+          '1.9.0': {
+            previous_version: null,
+            timestamp_installed: formatRFC3339(subMonths(new Date(), 4)),
+          },
+          '1.9.1': {
+            previous_version: '1.9.0',
+            timestamp_installed: formatRFC3339(subMonths(new Date(), 3)),
+          },
+          '1.9.2': {
+            previous_version: '1.9.1',
+            timestamp_installed: formatRFC3339(subMonths(new Date(), 2)),
+          },
+          '1.10.1': {
+            previous_version: '1.9.2',
+            timestamp_installed: formatRFC3339(new Date()),
+          },
+        },
+      },
+    };
+  });
```

Comment:
Would the `clients` handler be a better place for this since we are importing it into the sync handler already?

Reply:
Yup, didn't realize that initially. I'll move it there!